src/utils/strings.ts

```diff
@@ -10,6 +10,8 @@ export const trimAddress = (address?: string, chars = 4) => {
   return `${start}...${end}`;
 };
 
-const vowels = "eaiou".split("");
+const vowels = "eaiouh".split("");
```

Comment:
If this is only used for "a" vs "an" and only for English, I think not having `h` is correct.

Reply:
Its more complicated, but for simplicity I will remove "h".